target/linux/ramips/dts/mt7628an_wiznet_wizfi630s.dts

```diff
@@ -104,6 +104,16 @@
 		groups = "refclk";
 		function = "gpio";
 	};
+
+	p1led_an {
+		groups = "p1led_an";
+		function = "gpio";
+	};
+
+	p2led_an {
+		groups = "p2led_an";
+		function = "gpio";
+	};
```

Comment:
If we are touching this anyway, I'd prefer the syntax with one block as in your first attempt, just with the reduced number of entries. That one is used for the majority of devices. (Unless there is a specific reason for choosing this syntax here)